cmd/exporter/main.go

```diff
@@ -87,17 +81,30 @@ func export() error {
 		ArtifactsDir: artifactsDir,
 	}
 
-	factory, err := image.NewFactory(image.WithOutWriter(os.Stdout), image.WithEnvKeychain)
-	if err != nil {
-		return err
-	}
-
 	var stack metadata.StackMetadata
 	_, err = toml.DecodeFile(stackPath, &stack)
 	if err != nil {
 		outLog.Printf("no stack.toml found at path '%s', stack metadata will not be exported\n", stackPath)
 	}
 
+	if runImageRef == "" {
+		if stack.RunImage.Image == "" {
+			return cmd.FailCode(cmd.CodeInvalidArgs, "--run-image is required when stack.toml is not present in builder")
```

Comment:
The lifecycle flag is flag is `-image` rather than `--run-image`. Given that the `stack.toml` can have an arbitrary name if the path is supplied with a flag or an environment variable, perhaps we should say something more generic like "-image is required when there is no stack metadata available". Also, I'm not sure we should mention builders here, that feels like a platform specific concept.